lgsf/aws_lambda/handlers.py

```diff
@@ -13,25 +15,76 @@ def scraper_worker_handler(event, context):
     council = message["council"]
     command_name = message["scraper_type"]
     scraper_cls = load_scraper(council, command_name)
-    options = {"council": council, "verbose": True}
+    options = {"council": council, "verbose": True, "aws_lambda": True}
```

Comment:
FYI there are [documented environment vars in AWS Lambda](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-runtime) that might be useful to use